### PR TITLE
use ncc to build

### LINKED ISSUE #8: question: why keep node_modules on repo
if has `npm ci` command on the ci pipeline will install deps automatically. thx

### PR DESCRIPTION
**Pull request by sinchang**
1. use `@zeit/ncc` to build the source file, so that we can remove the `node_module` dir on the repo.

2. remove the useless package `@octokit/request`.

3. move the package `nock` to the devDependencies field.

closes #8

### CODE REVIEW

**Comment by rowi1de** [comment]
@sinchang nice cleanup.
You you try it the action works outside of this repo please?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ jobs:
     - uses: ./
       with: 
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        persons: "sinchang"
+        persons: "rowi1de"
```
Comment (rowi1de) [inline .github/workflows/test.yml]: To keep the possibility to test in own PRs
```suggestion
        persons: "rowi1de, sinchang"
```

**Comment by sinchang** [comment]
> @sinchang nice cleanup.
> You you try it the action works outside of this repo please?

sure. you can see here https://github.com/sinchang-bot/auto-assign-demo

File: .github/workflows/test.yml (hunk 1 above)
Comment (sinchang) [reply .github/workflows/test.yml]: oops... error `Review cannot be requested from pull request author.`
